Zscaler/zscaler-zia/ingest/parser.yml

```diff
@@ -133,9 +133,8 @@ stages:
         filter: "{{(json_event.message.event.ClientIP != null and json_event.message.event.ClientIP | is_ipaddress) or (json_event.message.event.clientip != null and json_event.message.event.clientip | is_ipaddress) or (json_event.message.event.clt_sip != null and json_event.message.event.clt_sip | is_ipaddress) or (json_event.message.event.csip != null and json_event.message.event.csip | is_ipaddress) or (json_event.message.event.sourceip != null and json_event.message.event.sourceip | is_ipaddress)}}"
 
       - set:
-          destination.ip: "{{json_event.message.event.srv_dip or json_event.message.event.sdip or json_event.message.event.destinationip}}"
-        filter: "{{(json_event.message.event.srv_dip != null and json_event.message.event.srv_dip | is_ipaddress) or (json_event.message.event.sdip != null and json_event.message.event.sdip | is_ipaddress) or (json_event.message.event.destinationip != null and json_event.message.event.destinationip | is_ipaddress)}}"
-
+          destination.ip: "{{json_event.message.event.srv_dip or json_event.message.event.sdip or json_event.message.event.destinationip or json_event.message.event.serverip}}"
+        filter: "{{(json_event.message.event.srv_dip != null and json_event.message.event.srv_dip | is_ipaddress) or (json_event.message.event.sdip != null and json_event.message.event.sdip | is_ipaddress) or (json_event.message.event.destinationip != null and json_event.message.event.destinationip | is_ipaddress) or (json_event.message.event.serverip != null and json_event.message.event.serverip | is_ipaddress)}}"
```

Comment:
```suggestion
        filter: "{{json_event.message.event.srv_dip | is_ipaddress
  or json_event.message.event.sdip | is_ipaddress
  or json_event.message.event.destinationip | is_ipaddress
  or json_event.message.event.serverip | is_ipaddress }}"
```

Reply:
`is_ipaddress` already checks for none

Reply:
I just based my code on what @squioc done before, didn't know but thanks!

Reply:
@PierrickV the YAML test didn't pass your changes, I wanted to regroup your changes on 1 unique line but I had another test failed (intake format one) and I admit that I don't understand why

Reply:
There was an orphan parenthesis at the beging of your code
```
filter: "{{(
-----------^
```